### ISSUE TITLE
Error when reader.readtext can't detect text

### ISSUE THREAD

**Issue by Johndirr** [JaidedAI/Easy_OCR#7]
When you run `reader.readtext ` on an image that has no text you get the following error:

```
File "C:\Users\blub\Envs\jaidedread\lib\site-packages\jaidedread\utils.py", line 443, in group_text_box
    x_min = min(mbox, key=lambda x: x[0])[0]
ValueError: min() arg is an empty sequence
```
Would be nice to have no error and empty return values instead.

**Comment by rkcosmos** [comment]
Agreed, will add that condition.

**Comment by rkcosmos** [comment]
fixed with #8
